### ISSUE TITLE
Стол+стол=стол

### ISSUE THREAD

**Issue by PersyFOX** [TauCetiStation/TauCetiClassic#10916]
#### Подробное описание проблемы
При установке стола на тайл с другим столом, первый стол удаляется
#### Что должно было произойти
Проверка наличия стола на тайле с персонажем, в следствие чего новый стол бы ставился или нет
#### Что произошло на самом деле
Новый стол уничтожает стол изначальный, просто удаляя его
#### Как повторить
Залезть на стол, поставить новый стол на тот же тайл
#### Дополнительная информация:
Проверял только с деревянными. Но помоему с металлическими та же история

**Comment by pro100kotah** [comment]
Да, с металлическими такая же тема. Можно на обычный стол поставить укрепленный и тот удалится. И наоборот вроде тоже.

**Comment by TauKitty** [comment]
Эта проблема была упомянута в **Tau Ceti Station Forum**. Там могут быть соответствующие подробности:

https://forum.taucetistation.org/t/bounty-rajskoe-naslazhdenie/46003/15
